node/node.d.ts

```diff
@@ -211,6 +211,23 @@ declare module NodeJS {
 
     export interface ReadWriteStream extends ReadableStream, WritableStream {}
 
+    export interface Events extends EventEmitter { }
+
+    export interface Domain extends Events {
+        run(fn: Function): void;
+        add(emitter: Events): void;
+        remove(emitter: Events): void;
+        bind(cb: (err: Error, data: any) => any): any;
+        intercept(cb: (data: any) => any): any;
+        dispose(): void;
+
+        addListener(event: string, listener: Function): Domain;
+        on(event: string, listener: Function): Domain;
+        once(event: string, listener: Function): Domain;
+        removeListener(event: string, listener: Function): Domain;
+        removeAllListeners(event?: string): Domain;
+    }
```

Comment:
please modify this line
https://github.com/cjbarth/DefinitelyTyped/blob/65b984206300e3cf8884ab07e46400a760aa733a/node/node.d.ts#L1983
likes the following pattern.
https://github.com/cjbarth/DefinitelyTyped/blob/65b984206300e3cf8884ab07e46400a760aa733a/node/node.d.ts#L458